src/site/content/en/learn/css/functions/functions.assess.yml

```diff
@@ -0,0 +1,54 @@
+setLeader: Test your knowledge of functions
+tabLabel: question
+questions:
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "2"
+    stem: CSS functions can be identified by which characters?
+    options:
+      - content: "`[]`"
+        rationale: "These characters are for arrays in Javascript"
+      - content: "`{}`"
+        rationale: "These characters wrap rules in CSS"
+      - content: "`()`"
+        rationale: "Functions use these characters to wrap arguments yep!"
+      - content: "`::`"
+        rationale: "These characters are for pseudo-elements in CSS"
+      - content: "`:`"
+        rationale: "These characters are for pseudo-classes in CSS"
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "0"
+    stem: CSS has built in math functions?
+    options:
+      - content: "True"
+        rationale: "There's a lot of them, and more being added to specs and
+        browsers!"
+      - content: "False"
+        rationale: "Try again!"
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "0"
+    stem: Many CSS functions can nest within each other.
```

Comment:
This question seems a bit broad -- perhaps do a specific example of nested functions? I.e. Is the following valid: `<code>`